packages/cli/src/commands/workflow/diagram/handleWorkflowDiagramResponse.ts

```diff
@@ -1,8 +1,8 @@
+import { getModelViewerHtml } from './getModelViewerHtml';
```

Comment:
Prefer to have external libs before project imports, can we do import fs and path before this?